service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -1081,37 +1085,51 @@ public Boolean status() {
   }
 
   /**
-   * Builds a policy list with a single provided owner and empty reader, writer and application
-   * policies.
+   * Builds a policy list with a single provided owner and empty reader, writer, project-owner, and
+   * application policies.
    *
    * <p>This is a helper function for building the policy section of a request to create a workspace
    * resource in Sam. The provided user is granted the OWNER role and empty policies for reader,
-   * writer, and application are also included.
+   * writer, project-owner, and application are also included.
    *
    * <p>The empty policies are included because Sam requires all policies on a workspace to be
    * provided at creation time. Although policy membership can be modified later, policy creation
    * must happen at the same time as workspace resource creation.
    */
-  private Map<String, AccessPolicyMembershipV2> defaultWorkspacePolicies(String ownerEmail) {
-    Map<String, AccessPolicyMembershipV2> policyMap = new HashMap<>();
+  private Map<String, AccessPolicyMembershipRequest> defaultWorkspacePolicies(
+      String ownerEmail, String projectOwnerGroupId) {
```

Comment:
Should `projectOwnerGroupId` be `@ Nullable` here as well?

Reply:
I forget about @ Nullable.  I will go back and add it all the way up.